pkg/apis/eventbus/v1alpha1/types.go

```diff
@@ -145,6 +145,7 @@ type NativeStrategy struct {
 type ContainerTemplate struct {
 	Resources       corev1.ResourceRequirements `json:"resources,omitempty" protobuf:"bytes,1,opt,name=resources"`
 	ImagePullPolicy corev1.PullPolicy           `json:"imagePullPolicy,omitempty" protobuf:"bytes,2,opt,name=imagePullPolicy,casttype=PullPolicy"`
+	SecurityContext corev1.SecurityContext      `json:"securityContext,omitempty" protobuf:"bytes,3,opt,name=securityContext"`
```

Comment:
Use pointer please.

Reply:
Also add the logic in controller code https://github.com/argoproj/argo-events/blob/master/controllers/eventbus/installer/nats.go#L708-L751

Reply:
Done.